cli/cmd/install.go

```diff
@@ -69,6 +69,8 @@ type (
 		WebResources *resources
 
 		Identity *installIdentityValues
+
+		CATrust *caTrustValues
```

Comment:
i'm a little out of context, can you explain the motivation for separate `KeyPEM`/`CertPem` values for the MWC/VWC and the identify system values in `installIdentityValues`? or, taking this a step further, re-using the `linkerd-identity-issuer` secret?

Reply:
Based on my brief chat with @olix0r, we prefer to keep them separated, because they exist for different purposes. `installIdentityValues`, as we all know, is used for data plane mTLS and consist of fields like issuance life time, clock skew allowances etc. that aren't relevant to the CA. The CA is used to sign certs that will be used by webhooks and taps to authenticate with the k8s API Server.